BSP/Simulator/DataGeneration/Fans.py

```diff
@@ -0,0 +1,27 @@
+import csv
+import os
+import fcntl
+import config
+
+#This module will hold the statuses of the Fans on the BPS.
+
+#path name to file
+file = config.directory_path + config.files['Fans']
+
+#returns integer representing duty cycle
+def read():
+    duty = []
+    os.makedirs(os.path.dirname(file), exist_ok=True)   # creates directory if not exists
+    with open(file, 'a') as csvfile:
+        pass    # creates file if not exists
+    with open(file, 'r') as csvfile: #read file
+        fcntl.flock(csvfile.fileno(), fcntl.LOCK_EX)    # Lock file
+        csvreader = csv.reader(csvfile)
+        for row in csvreader:
+            duty.append(row)
+        fcntl.flock(csvfile.fileno(), fcntl.LOCK_UN)    # Unlock file
+        if len(duty):
+            return int(duty[0][0])
```

Comment:
Right now, this looks like it only gets the duty cycle of the first fan. We should have a way to read each fan's duty cycle in the simulator.